spec/console_display_spec.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "../lib/console_display"
+require_relative "../lib/board"
+require "rspec"
+require "stringio"
```

Comment:
Good idea requiring `StringIO`. However it's not used. How can we use it to move away from setting expectations on things that we don't own (`STDOUT` & `STDIN`)?